docs/modules/ROOT/pages/operations.adoc

```diff
@@ -339,12 +340,12 @@ The Bolt+routing feature on Neo4j drivers discovers a cluster member (drivers ha
 
 | `editorLint`
 | `false`
-| TODO
+| Enable/disable squiggly lines under text.
```

Comment:
This is what the setting is supposed to do, but it doesn't work

Reply:
same as below